slsa_for_models/README.md

```diff
@@ -0,0 +1,85 @@
+# SLSA for Models
+
+To protect the supply chain of traditional software against tampering (like in
+the [Solarwinds attack][solarwinds]), we can generate SLSA provenance, for
+example by using the [SLSA L3 GitHub generator][slsa-generator].
+
+This project shows how we can use the [SLSA L3 GitHub generator][slsa-generator]
+to generate SLSA provenance for ML models. The SLSA generator was originally
+developed for traditional software to protect against tampering with builds,
+such as in the [Solarwinds attack][solarwinds], and this project is a proof of
+concept that the _same supply chain protections can be applied to ML_.
+
```

Comment:
I think we should add one line about that the verification of provenance gives you. You're guaranteed the training was done in the repo at a particular commit sha / tag.

You can use `--print-provenance` option to see the output of provenance if needed. I'd simply link to https://github.com/slsa-framework/slsa-verifier#artifacts for further information.